Formula/c/cucumber-cpp.rb

```diff
@@ -1,8 +1,9 @@
 class CucumberCpp < Formula
   desc "Support for writing Cucumber step definitions in C++"
   homepage "https://cucumber.io"
-  url "https://github.com/cucumber/cucumber-cpp/archive/refs/tags/v0.6.tar.gz"
-  sha256 "d4f8155b665a8b338a60f97bd652bb04a1b41f5c32750e13dbf48664a942d93a"
+  url "https://github.com/cucumber/cucumber-cpp.git",
+      tag:      "v0.7.0",
+      revision: "ceb025fb720f59b3c8d98ab0de02925e7eab225c"
```

Comment:
Switched to git urls because upstream's build added assumption that it's built out of a Git repo: https://github.com/cucumber/cucumber-cpp/commit/8a0d16b8f59dd1c9d75045aa016d9ce235257311